tests/e2e/cjs/index.test.js

```diff
@@ -277,3 +290,37 @@ describe('ANTRegistry', async () => {
     assert(Array.isArray(affiliatedAnts.Controlled));
   });
 });
+
+describe('Signing', async () => {
```

Comment:
related to test structure this `describe` is inaccurate. we're really just confirming the signers provided to IO are supported, so I wouldn't place the respective `its` under existing `desribes` for each top level class we export